libr/core/cmd_print.c

```diff
@@ -4840,6 +4840,38 @@ static int cmd_print(void *data, const char *input) {
 		case 'J': // pdJ
 			formatted_json = true;
 			break;
+		case 'u': // pdu
+			processed_cmd = true;
+			if (input[2] == '?') {
+				//todo - print help for pdu/pdub/pdur
+			} else if (input[2] == 'b' ){ // pdub
+				//todo - define pdub
+			} else if (input[2] == 'r') { //pdur
+				//todo - define pdur
+			} else { // pdu
+				ut64 start_addr = core->offset;
+				ut8 *buf = NULL;
+				if (strlen(input) < 3){
```

Comment:
missing space before (

Reply:
Added missing space to `if (strlen (input) < 3)`.